src/components/select-menu/SelectMenu.spec.tsx

```diff
@@ -0,0 +1,220 @@
+import * as React from 'react';
+import {render, waitFor, fireEvent, within} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import SelectMenu from './SelectMenu';
+
+export function RenderSelectMenu(props: {
+  multiSelect?: boolean;
+  defaultExpanded?: boolean;
+  disabled?: boolean;
+  valid?: boolean;
+  invalid?: boolean;
+}) {
+  const [selectedOptions, setSelectedOptions] = React.useState([]);
+  const handleOptionChange = option => {
+    if (props.multiSelect) {
+      if (!selectedOptions.find(item => item.value === option.value))
+        // If it was not present, add it
+        setSelectedOptions([...selectedOptions, option]);
+      else {
+        // else, remove from array of selected options
+        setSelectedOptions(
+          [...selectedOptions].filter(item => item.value !== option.value)
+        );
+      }
+    } else {
+      setSelectedOptions([option]);
+    }
+  };
+
+  return (
+    <SelectMenu
+      options={[
+        {
+          value: 'physics',
+          label: 'Physics',
+        },
+        {value: 'history', label: 'History'},
+        {value: 'science', label: 'Science'},
+      ]}
+      placeholder="Select..."
+      selectedOptions={selectedOptions}
+      onOptionChange={handleOptionChange}
+      disabled={props.disabled}
+      valid={props.valid}
+      invalid={props.invalid}
+      {...props}
+    />
+  );
+}
+
+window.HTMLElement.prototype.scroll = jest.fn();
+
+describe('<SelectMenu />', () => {
+  it('renders SelectMenu in the default state', () => {
+    const select = render(<RenderSelectMenu />);
+    const selectElement = select.getByRole('combobox') as HTMLElement;
+
+    expect(select.getByText('Select...')).toBeTruthy();
+    expect(selectElement.getAttribute('aria-expanded')).toBe('false');
+    expect(selectElement.getAttribute('aria-disabled')).toBeFalsy();
```

Comment:
As a general rule we should be as explicit as possible. Please don't overuse `toBeFalsy` and `toBeTruthy` and use `toBe`, `toBeUndefined`, `toEqual`, `toHaveAttribute` etc.

In this specific case - testing `aria-disabled` feels inconsitend across all tests.